Content.Server/Singularity/EntitySystems/GravityWellSystem.cs

```diff
@@ -27,30 +25,29 @@ public sealed class GravityWellSystem : SharedGravityWellSystem
 #endregion Dependencies
 
     /// <summary>
-    /// The minimum range at which gravpulses will act.
+    /// The minimum range at which gravscans will act.
     /// Prevents division by zero problems.
     /// </summary>
-    public const float MinGravPulseRange = 0.00001f;
+    public const float MinGravWellRange = 0.00001f;
 
     public override void Initialize()
     {
         base.Initialize();
         SubscribeLocalEvent<GravityWellComponent, ComponentStartup>(OnGravityWellStartup);
 
         var vvHandle = _vvManager.GetTypeHandler<GravityWellComponent>();
-        vvHandle.AddPath(nameof(GravityWellComponent.TargetPulsePeriod), (_, comp) => comp.TargetPulsePeriod, SetPulsePeriod);
+        vvHandle.AddPath(nameof(GravityWellComponent.TargetScanPeriod), (_, comp) => comp.TargetScanPeriod, SetScanPeriod);
     }
 
     public override void Shutdown()
     {
         var vvHandle = _vvManager.GetTypeHandler<GravityWellComponent>();
-        vvHandle.RemovePath(nameof(GravityWellComponent.TargetPulsePeriod));
+        vvHandle.RemovePath(nameof(GravityWellComponent.TargetScanPeriod));
         base.Shutdown();
     }
 
     /// <summary>
-    /// Updates the pulse cooldowns of all gravity wells.
-    /// If they are off cooldown it makes them emit a gravitational pulse and reset their cooldown.
+    /// Updates the set of captured entities for all gravity wells.
     /// </summary>
     /// <param name="frameTime">The time elapsed since the last set of updates.</param>
     public override void Update(float frameTime)
```

Comment:
Looks like someone added IsFirstTimePredicted here but it doesn't even do anything on the server.

Reply:
Removed. I honestly just cargo-culted it when I was told to switch to target times and then didn't remove it when I found out how it worked.

Reply:
Added back because it's now in shared.